templates/users/new.rs.html

```diff
@@ -15,9 +15,9 @@ <h1>@i18n!(ctx.1, "Create an account")</h1>
             @input!(ctx.1, password (password), "Password", form, errors.clone(), "minlenght=\"8\"")
             @input!(ctx.1, password_confirmation (password), "Password confirmation", form, errors, "minlenght=\"8\"")
 
-            <input type="submit" value="@i18n!(ctx.1, "Create account")" />
+            <input type="submit" value="@i18n!(ctx.1, "Create your account")" />
         </form>
     } else {
-        <p class="center">@i18n!(ctx.1, "Sorry, but registrations are closed on this instance. Try to find another one")</p>
+<p class="center">@i18n!(ctx.1, "Apologies, but registrations are closed on this instance. Visit joinplu.me/#instances, to find a different one")</p>
```

Comment:
It is a good idea to redirect to joinplu.me, but as it is the link won't work. Could you replace it with something like:

```html
<p class="center">@i18n!(ctx.1, "Apologies, but registrations are closed on this instance. Visit joinplu.me/#instances, to find a different one")</p>
<p class="center">
    <a href="https://joinplu.me/#instances">@i18n!(ctx.1, "Find a different one")</a>
</p>
```

to make it clickable, please? (no problem if you don't, it is still better than before)